src/content/1.5/code/agda/snippet21.agda

```diff
@@ -0,0 +1,2 @@
+inj₁ : A → C
+inj₂ : B → C
```

Comment:
I wonder if preserving naming from Haskell would make sense here:
```agda
i : a -> c
j : b -> c
```
or showing names from stdlib. Both seems like valid approaches.

Reply:
Yeah, good point.  I can't decide whether it's better to be faithful to the original text or expose the reader to idiomatic Agda.

Reply:
changed `inj₁ to `i` and `inj₂` to `j`